imgui.go

```diff
@@ -686,3 +686,62 @@ func IsItemHovered() bool {
 func IsKeyPressed(key int) bool {
 	return C.iggIsKeyPressed(C.int(key)) != 0
 }
+
+// BeginColumns calls BeginColumnsV().
+func BeginColumns(count int, label string) {
```

Comment:
The terse variant of `Columns()` has no parameters. All three arguments have default arguments.